projects/plugins/jetpack/modules/masterbar/admin-menu/upsell-nudge.js

```diff
@@ -0,0 +1,85 @@
+( () => {
+	function requestJitm() {
+		if ( window.wpApiSettings.root.indexOf( 'https://public-api.wordpress.com/' ) === 0 ) {
+			return wp.apiFetch( {
+				apiNamespace: 'rest/v1.1',
+				global: true,
+				path:
+					'/jetpack-blogs/' +
+					window._currentSiteId +
+					'/rest-api?path=/jetpack/v4/jitm&query={"message_path":"calypso:sites:sidebar_notice"}',
+				method: 'GET',
+			} );
+		}
+
+		return wp.apiFetch( {
+			method: 'GET',
+			url: window.wpApiSettings.root + 'jetpack/v4/jitm?message_path=calypso:sites:sidebar_notice',
+		} );
+	}
+
+	requestJitm().then( response => {
+		let data;
+		if ( response.data ) {
```

Comment:
This is another polymorphic behavior:

* For WP.com simple sites, the WP.com REST API returns enveloped responses, and the response is contained inside a `data` field.
* For Jetpack / Atomic sites, the built-in REST API endpoint returns the data raw, in an array.